fishtank/src/cluster.ts

```diff
@@ -51,20 +57,53 @@ export class Cluster {
       labels: { [CLUSTER_LABEL]: this.name },
     }
 
+    const args: string[] = []
+
     if (options.config) {
       const configString = JSON.stringify(options.config)
 
-      const dest = join(tmpdir(), 'fishtank', containerName, '.ironfish')
+      const dest = join(tmpdir(), 'fishtank', this.name, options.name, '.ironfish')
       await promises.mkdir(dest, {
         recursive: true,
       })
```

Comment:
This logic is duplicated with the code below. Consider grouping it like this:

```typescript
if (options.config || options.networkDefinition) {
  // create directories...
  // add volumes...
}

if (options.config) {
  // write config.json
}

if (options.networkDefinition) {
  // write network.json
}

Reply:
added